README.md

```diff
@@ -41,3 +41,11 @@ gunicorn and django-admin commands
 ## CONFIGURATION
 
 Use `PYCCW_LOG_LEVEL=(DEBUG|...)` to configure log levels.
+
+## CONTRIBUTING
+
+PRs are welcome! Please add yourself to the contributors list in the PR.
+
+### Contributors
+
```

Comment:
Add your profile too and I will be happy :)
```markdown
* alex <alex@pdp7.net> (author)
```

Reply:
Done!

Reply:
Nice! Congrats and thanks for the award ;)